Foundation/NSURL.swift

```diff
@@ -32,7 +32,11 @@ private func _standardizedPath(_ path: String) -> String {
     if !path.absolutePath {
         return path._nsObject.standardizingPath
     }
+    #if os(Windows)
+    return String(path.map({$0 == "/" ? "\\" : $0}))
```

Comment:
My understanding is that the standardizing should collapse relative components (i.e. `..` should be chased, `.` dropped).  As such, any reason to not just use `PathCchCanonicalizeEx`?